collector/collector.go

```diff
@@ -56,21 +57,26 @@ func main() {
 	}
 
 	ctx := context.Background()
-	l, err := ctlog.New(*logURL, *logName, *b64PubKey)
+	l, err := ctlog.New(*logURL, *logName, *b64PubKey, 24*time.Hour)
```

Comment:
Could you move this to a const? It's hard to tell at a glance what this duration controls, without going and looking up the docs for `ctlog.New()`.

Reply:
Ah yes, I did actually mean to move this to a flag, which I have now done.